apps/api/src/app/graphql.middleware.ts

```diff
@@ -0,0 +1,23 @@
+import { FieldMiddleware, NextFn, MiddlewareContext } from '@nestjs/graphql'
+
+export const maskOutFieldsMiddleware: FieldMiddleware = async (
```

Comment:
Would be nice to get tests for this. At least a unit test.

Integration test for IdentityPerson field filtering would also be nice.